data/projects/vulture.md

```diff
@@ -1,57 +1,126 @@
+We would need to analyse every construct individually on a case by case basis. For example, we already know that import statements can be predicted with 100% surety (except for * imports, where it would be 0%), but functions often have false-positives.
+
+The confidence value will be alike the ones given below: (The finer grained distinctions will need further discussion)
+
+- `import` -->  `100%`
+- `from foo import *` --> `0%`
+- `variable` --> `<100%`
+- `function` --> `<100%`
+- `class` --> `<100%`
+- `if False` --> `100%`
 
 References:
 
-- [vulture](https://bitbucket.org/jendrikseipp/vulture)
+- [vulture](https://github.com/jendrikseipp/vulture)
+- [Project Proposal](https://docs.google.com/document/d/1gzRH-rdJsiAD-TOEB0O4OqtcnTp0zotcNl6agAuXGHw/edit?usp=sharing)
 
 #### Milestones
 
-##### PREPARATION/BONDING
+##### Pre - Community Bonding Period ( Up Till  May 4)
+- Initial investigation and discussion on what features to be added to API, and start implementing them on the go.
+- Continue discussion with the community on ast and enhanced pyflakes ast - What are the underlying differences, and what can be possible implementations for source range acquisition.
+- Diagnose possible cases which would render the code unreachable.
+- Inculcate a habit of downloading the trending project of the day, running vulture on it, analyzing the results and adding anything suitable to whitelist file. This would help us in maintaining a rich whitelist file.
+- Prioritize all the goals and targets
+
+##### Community Bonding (May 5 - 30)
+- Finalize what should be implemented for source range acquisition on the basis of discussions during pre - community bonding period.
+
+##### Week 1 & 2 (May 31 - June 13)
+- Implement API - At this point, we would have a fully functioning API, and we would be ready to incorporate these changes into the VultureBear.
+- Refactor VultureBear to incorporate API - This would be relatively easy as the API would have had until now respected it’s usage in the VultureBear.
+
+##### Week 3 (June 14 - 20)
+- Realize VultureBear completely - At this point, it should be able to execute vulture, parse output, produce the patch file for removing the dead code what so ever found.
+Review the tests written so far and add relevant test cases.
+- Document Bear
+- API documentation including, but not limited to possible use cases, example code snippets
+- See that code is completely documented in place
+- Buffer time to finish pending work.
 
-- A concept for the source range acquisition is finalized.
+##### Week 4 (June 20 - 26)
+- Develop an analysis report on what all constructs can we detect with complete surety,  partial surety and the ones we cannot detect for the sake of implementing a confidence value with results. (A crude version is here in this thread).
+- Discuss on the levels of certainty we would need to have (perhaps four for highly accurate (100%), accurate (>80%), medium (70 - 80%) and can’t say)
 
-##### CODING PHASE 1
```

Comment:
could you keep the phases section in the document, and put the relevant weeks inside them